python/ray/_private/log.py

```diff
@@ -1,6 +1,9 @@
 import logging
 import threading
 from typing import Union
+import time
+
+LOG_KEY_TIMESTAMP_NS = "timestamp_ns"
```

Comment:
Why do we need to define `LOG_KEY_TIMESTAMP_NS` here instead of using the one in `logging_config.py`? Are there any circular dependencies?

Reply:
Yea, all those code runs in init, we can't import `LogKey` before all modules are fully initialized. The one way to break the circular dependencies is to define this first and reuse it everywhere

Reply:
Got it. If @jjyao is okay with this, I am fine with it.

Reply:
Is it possible to move the import to a local import within the `_setup_log_record_factory` function to avoid circular dependencies, if any?

Reply:
Not possible, due to this was call when Ray is imported and we can't import partially imported modules

Reply:
But to be honest, we probably don't care about `timestamp_ns` being in `LogKey` or not, since it's not used anywhere expect for setting up log record factory here. Maybe we can just drop it from `LogKey` altogether if both of you don't mind